infrastructure/zk/src/docker.ts

```diff
@@ -138,6 +140,10 @@ export async function build(image: string, cmd: Command) {
     await dockerCommand('build', image, cmd.customTag);
 }
 
+export async function customBuildForHyperchain(image: string, imageName: string) {
+    await dockerCommand('build', image, '', false, imageName);
```

Comment:
Nit: `image` and `imageName` look confusing here, probably should be `image` and `dockerOrg` (or at least `hyperchainName`).

Reply:
ops. Totally